src/main/java/org/ow2/proactive/catalog/service/CatalogObjectService.java

```diff
@@ -165,12 +165,8 @@ public CatalogObjectMetadata updateObjectMetadata(String bucketName, String name
             throw new WrongParametersException("at least one parameter should be present");
         }
         CatalogObjectEntity catalogObjectEntity = catalogObjectRevisionEntity.getCatalogObject();
-        if (kind.isPresent()) {
-            catalogObjectEntity.setKind(kind.get());
-        }
-        if (contentType.isPresent()) {
-            catalogObjectEntity.setContentType(contentType.get());
-        }
+        kind.ifPresent(k -> catalogObjectEntity.setKind(k));
+        contentType.ifPresent(ct -> catalogObjectEntity.setContentType(ct));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)